src/main/java/game/race/Car.java

```diff
@@ -4,26 +4,49 @@
 import java.util.List;
 import java.util.Random;
 
+import game.race.dto.RaceDto;
+
 public class Car {
 
     private static final int THRESHOLD = 4;
 
+    private String name;
     private int moveCount;
 
+    public Car() {}
```

Comment:
기본 생성자를 호출하고 getName()을 호출할 수 있을 것 같네요
아래처럼 해줘야 안전할 것 같습니다.
```suggestion
    public Car() {
        this("");
    }
```